guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,227 @@
+[id="upgrading-{project-context}-or-{smart-proxy-context}-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL-abbr} 9 in-place by using Leapp
+
+{Project} and {SmartProxy} are supported on both {EL} 8 and {EL} 9.
+You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9. 
+When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to upgrade as well as to help detect and resolve issues that could prevent you from upgrading successfully.
+
+.Prerequisites
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
+ifdef::foreman-el,foreman-deb,katello[]
+* Review upgrade warnings before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}[Release notes].
+endif::[]
+* {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+ifndef::satellite[]
+* Access to available repositories or a local mirror of repositories.
+endif::[]
+ifeval::["{mode}" == "disconnected"]
+* You require access to {RHEL} and {Project} packages.
+Obtain the ISO files for {RHEL}{nbsp}9 and {Project}.
+For more information, see xref:Upgrading_satellite_upgrading-disconnected[].
```

Comment:
It would be better to use the link to the particular section in the Installing disconnected guide as it is done in step 8. of [this procedure](https://theforeman-foreman-documentation-preview-pr-3070.surge.sh/nightly/Upgrading_Project_Disconnected/index-satellite.html#upgrading_a_disconnected_satellite_server_upgrading-disconnected).

By referring the user to the top of this guide, we're not really helping them.